### PR TITLE
added_homework11

### PR DESCRIPTION
**Pull request by AliKonoplenko**
HW 11 - using task command
```
with command task() https://docs.cypress.io/api/commands/task
create in cypress.config file command that save into json file urls
use that task() command in cy.js files for saving the url in 
can use package 'fs-extra' or other 
```

HW 12 -  using page objects
```
create new page object for page of website 'dou.ua'
based on example https://github.com/Hypercubed/cypress-page-object-pattern/tree/master# page-module-2
create new test file with Cypres framework with selectors/methods from created page object
```

### CODE REVIEW

File: cypress/e2e/homeWork/homework/homework11.cy.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+///<reference types="cypress" />
+
+
+context('Check the AIN website', () => {
+
+    it('Open the main page and click startups', () => {
+      cy.visit('https://ain.ua/')
+        cy.task('saveURL', 'https://ain.ua/')
```
Comment (Densf2) [inline cypress/e2e/homeWork/homework/homework11.cy.js:8]: should you code work with cy.url()?

File: cypress/pages/main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export default class MainPage {
```
Comment (Densf2) [inline cypress/pages/main.js]: the task to create page object based on js object - not on class.
could you update your code

File: cypress/e2e/homeWork/homework/homework11.cy.js (hunk 1 above)
Comment (AliKonoplenko) [reply cypress/e2e/homeWork/homework/homework11.cy.js:8]: yeap, added new commit, check it out, please

File: cypress/pages/main.js (hunk 2 above)
Comment (AliKonoplenko) [reply cypress/pages/main.js]: updated the code (based on js object) - homework 12

File: cypress/e2e/homeWork/homework/homework11.cy.js (hunk 1 above)
Comment (Densf2) [reply cypress/e2e/homeWork/homework/homework11.cy.js:8]: you can use method cy.url() iniside your code
like this `cy.task('saveURL', cy.url())`

**Pull request merged**